clash-prelude/src/Clash/Explicit/BlockRam.hs

```diff
@@ -247,62 +256,62 @@ replace it with the string @\"undefined\"@ in the first few leading outputs.
 
 Finally we get to using 'blockRam'. On FPGAs, 'Clash.Prelude.RAM.asyncRam' will
 be implemented in terms of LUTs, and therefore take up logic resources. FPGAs
-also have large(r) memory structures called /Block RAMs/, which are preferred,
+also have large(r) memory structures called /block RAMs/, which are preferred,
 especially as the memories we need for our application get bigger. The
-'blockRam' function will be translated to such a /Block RAM/.
+'blockRam' function will be translated to such a /block RAM/.
 
-One important aspect of Block RAMs have a /synchronous/ read port, meaning that,
-unlike the behavior of 'Clash.Prelude.RAM.asyncRam', given a read address @r@
-at time @t@, the value @v@ in the RAM at address @r@ is only available at time
-@t+1@.
+One important aspect of block RAMs is that they have a /synchronous/ read port,
+meaning that, unlike the behavior of 'Clash.Prelude.RAM.asyncRam', given a read
```

Comment:
* I would just say "meaning " instead of "meaning that, " with the extra comma. Having those two commas gives unnecessary emphasis on the words between them

  * I would take the oppourtunity to re-work the end of this sentence too. We give names to the address and value (`r` and `v`) but the only important bit is time IMO. Something like this reads better to me:

    > ... meaning, unlike an asyncRam, a value read at time t becomes available at cycle t + 1.